core/src/main/java/org/apache/gravitino/storage/relational/mapper/CatalogMetaSQLProviderFactory.java

```diff
@@ -49,6 +52,88 @@ static class CatalogMetaMySQLProvider extends CatalogMetaBaseSQLProvider {}
 
   static class CatalogMetaH2Provider extends CatalogMetaBaseSQLProvider {}
 
+  static class CatalogMetaPostgreSQLProvider extends CatalogMetaBaseSQLProvider {
```

Comment:
Can we have a package "postgresql" under this and move all the pg class to that folder.

Besides, we have several changes for PG, but we don't have UTs to cover the pg support, only have IT. I think it is not enough, we should add UTs for PG, as well as for MySQL.

Reply:
> Can we have a package "postgresql" under this and move all the pg class to that folder.

Okay.

> Besides, we have several changes for PG, but we don't have UTs to cover the pg support, only have IT. I think it is not enough, we should add UTs for PG, as well as for MySQL.

Yeah, I will try to boost a separate PR to test the MySQL and PG backend in UT.